FSJ_django20_project/FSJ/templates/FSJ/base_with_banner.html

```diff
@@ -1 +1 @@
-{% extends "FSJ/base.html" %}
{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}

{% block banner %}
    <div class="banner">
        <div>
            {# Translators: Home link on the main banner #}
            <a class="header" href='/home'>{% trans "Home" %}</a>
        </div>
        <div>
            {# Translators: Profile link on the main banner #}
            <a class="header" href='/profile'>{% trans "Profile" %}</a>
        </div>
        {% if FSJ_user.user_class == "Coordinator" %}
        <div>        
            {# Translators: Students link on the main banner #}
            <a class="header" href='/studentlist'>{% trans "Students" %}</a>
        </div>
        <div>        
            {# Translators: Adjudicators link on the main banner #}
            <a class="header" href='/adjudicatorlist'>{% trans "Adjudicators" %}</a>
        </div>
        {% endif %}
        <div>
            {# Translators: Logout link on the main banner #}
            <a class="header" href='/logout'>{% trans "Logout" %}</a>
        </div>

    </div>
{% endblock %}
\ No newline at end of file
+{% extends "FSJ/base.html" %}
{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}

{% block banner %}
    <div class="banner">
        <div>
            {# Translators: Home link on the main banner #}
            <a class="header" href='/home'>{% trans "Home" %}</a>
        </div>
        <div>
            {# Translators: Profile link on the main banner #}
            <a class="header" href='/profile'>{% trans "Profile" %}</a>
        </div>
        {% if FSJ_user.user_class == "Coordinator" %}
        <div>        
            {# Translators: Students link on the main banner #}
            <a class="header" href='/studentlist'>{% trans "Students" %}</a>
        </div>
        <div>        
            {# Translators: Adjudicators link on the main banner #}
            <a class="header" href='/adjudicatorlist'>{% trans "Adjudicators" %}</a>
        </div>
        <div>        
            {# Translators: Awards link on the main banner #}
            <a class="header" href='/coord_awardslist'>{% trans "Awards" %}</a>
        </div>
        {% endif %}
        <div>
            {# Translators: Logout link on the main banner #}
            <a class="header" href='/logout'>{% trans "Logout" %}</a>
        </div>

    </div>
{% endblock %}

```

Comment:
The Awards view handler suggests Students can access the awards page (or some type of awards page) but the hyperlink in the banner is hidden to all but Coordinators? How are Students intended to access this?

Reply:
I believe we will have a separate awards page for students right? Right now, clicking the awards header in the banner will call "award" in views.py. This will return a different response depending on the role of whoever is logged in. Right now, only the coordinators awards page is implemented